src/SLCore.UnitTests/NodeJS/Notifications/UnsupportedNodeVersionNotificationServiceTests.cs

```diff
@@ -21,6 +21,7 @@
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.Notifications;
 using SonarLint.VisualStudio.SLCore.NodeJS.Notifications;
+using Language = SonarLint.VisualStudio.SLCore.Common.Models.Language;
```

Comment:
Was this needed?

Reply:
It was automatically applied on commit because of the "Optimize imports" setting on Rider